src/components/Calculator.js

```diff
@@ -14,7 +14,7 @@ class Calculator extends React.Component {
   handleClick = (event) => {
     const res = calculate(this.state, event.target.textContent);
     this.setState(res);
-  }
+  };
 
   render() {
     const { total, next } = this.state;
```

Comment:
- [ ] According to the project's requirements kindly make sure The Calculator component (components/Calculator.js)  is refactored from a **class-based component** to a **functional component** using [hooks](https://reactjs.org/docs/hooks-intro.html). **And for best practices, I suggest you use the ES6 syntax `(the arrow function)` instead of the normal function as I mentioned in the previous comment**
Once, you refactored the `Calculator.js` component make sure the app is still working as expected.

Reply:
ES6 arrow functions refactored.